qa-test-app/src/main/java/com/mapbox/navigation/qa_test_app/view/RetainedActiveGuidanceFragment.kt

```diff
@@ -1,97 +0,0 @@
-package com.mapbox.navigation.qa_test_app.view
-
-import android.os.Bundle
-import android.view.LayoutInflater
-import android.view.View
-import android.view.ViewGroup
-import androidx.fragment.app.Fragment
-import androidx.lifecycle.asLiveData
-import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
-import com.mapbox.navigation.core.MapboxNavigation
-import com.mapbox.navigation.core.lifecycle.MapboxNavigationApp
-import com.mapbox.navigation.core.lifecycle.MapboxNavigationObserver
-import com.mapbox.navigation.core.trip.session.TripSessionState
-import com.mapbox.navigation.dropin.extensions.attachStarted
-import com.mapbox.navigation.dropin.extensions.flowRoutesUpdated
-import com.mapbox.navigation.dropin.extensions.flowTripSessionState
-import com.mapbox.navigation.qa_test_app.databinding.FragmentActiveGuidanceBinding
-import com.mapbox.navigation.qa_test_app.lifecycle.bottomsheet.DropInTripProgress
-import com.mapbox.navigation.qa_test_app.lifecycle.topbanner.DropInManeuver
-import kotlinx.coroutines.flow.combine
-import kotlinx.coroutines.flow.distinctUntilChanged
-import kotlinx.coroutines.flow.onStart
-
-@OptIn(ExperimentalPreviewMapboxNavigationAPI::class)
-class RetainedActiveGuidanceFragment : Fragment() {
```

Comment:
@tomaszrybakiewicz Will you restore this Fragment?

It is serving as a way to ensure our drop in architecture does not have memory leaks with retained fragments.

It is also an example for `MapboxNavigationApp.attach(this)` working with Fragments

Reply:
@abhishek1508 What would you like to do about this?

Reply:
@kmadsen Where are we using this fragment exactly? I didn't see it anywhere and hence found it reasonable to be removed.

Reply:
It's in the `AppLifecycleActivity`. That activity is demonstrating the usage of Components _without_ a `NavigationView`

Reply:
@tomaszrybakiewicz It looks like `AppLifecycleActivity` and `RetainedActiveGuidanceFragment` are being used. We need to bring these 2 back and layout files that they are using. Sorry for the confusion.